src/dmap_import/util_rds.py

```diff
@@ -255,9 +290,38 @@ def truncate_table(
         self.execute(sa.text(f"{truncate_query};"))
 
         # Execute VACUUM to avoid non-deterministic behavior during testing
-        with self.session.begin() as cursor:
-            cursor.execute(sa.text("END TRANSACTION;"))
-            cursor.execute(sa.text(f"VACUUM (ANALYZE) {truncat_as};"))
+        self.vaccuum_analyze(table_to_truncate)
+
+
+def copy_local_to_db(
+    local_path: str, destination_table: DeclarativeBase
+) -> None:
+    """
+    Load local file into DB using psql COPY command
+    will throw if psql command does not exit with code 0
+
+    :param local_path: path to local file that will be loaded
+    :param destination_table: SQLAlchemy table object for COPY destination
+    """
+    with gzip.open(local_path, "rt") as gzip_file:
+        local_columns = gzip_file.readline().strip().lower().split(",")
+
+    copy_command = (
+        f"\\COPY {destination_table.__table__} "
+        f"({','.join(local_columns)}) "
+        "FROM PROGRAM "
+        f"'gzip -dc {local_path}' "
+        "WITH CSV HEADER"
+    )
+
+    psql = [
+        "psql",
+        f"postgresql://{create_db_connection_string()}",
+        "-c",
+        f"{copy_command}",
+    ]
+
+    subprocess.run(psql, check=True)
```

Comment:
we want to report back if this is copy is successful or not, which i'm assuming you can do via the return code.

Reply:
added logging to this function